cpp/include/cugraph_c/layout_algorithms.h

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2025, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <cugraph_c/error.h>
+#include <cugraph_c/graph.h>
+#include <cugraph_c/graph_functions.h>
+#include <cugraph_c/random.h>
+#include <cugraph_c/resource_handle.h>
+
+/** @defgroup layout Layout algorithms
+ */
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+
+/**
+ * @brief   Force Atlas 2
+ *
+ * NOTE: This currently wraps the legacy force atlas 2 implementation and is only
```

Comment:
This is inconsistent with the C API general approach.  Specifically, it exposes assumptions about the internal implementation through the API.  We've generally tried to avoid this.  If we were to ever create a MG version or to refactor FA to use graph primitives these assumptions would no longer be true.

I'll highlight some ideas on changes to make things more consistent.

Reply:
Got it. I have seen this line`NOTE: This currently wraps...implementation and is only
 * available in Single GPU implementation` in several legacy C code (`community_algorithms.h`) and I was under the assumption that we should add this note